modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -1233,6 +1233,36 @@ Use `DATABASE *` without the parameter to revoke the privilege on all databases.
 
 Topology notifications provide additional information related to managing databases and servers.
 
+[#_neo_clientnotification_cluster_nodatabasesreallocated]
+=== NoDatabasesReallocated
+
+.Notification details
+[cols="<1s,<4"]
+|===
+|Code
+m|Neo.ClientNotification.Cluster.NoDatabasesReallocated
+|Title
+a| `<command>` has no effect.
+|Severity
+m|INFORMATION
+|Category
+m|TOPOLOGY
+|===
+
+.Reallocating databases resulted in no allocation changes 
+====
+Command::
++
+[source, cypher]
+----
+REALLOCATE DATABASES
+----
+
+Description of the returned code::
+No databases were reallocated.
```

Comment:
Do we want to provide some further information, e.g. around that this is because the allocation already is stable/have converged (not sure what the correct term is)? Because as a user I would like to know if I e.g. should try again or if it was something that went wrong or if it is optimal already

Reply:
I agree with Louise. I think it would be very helpful with one or more examples of when this can happen and how to "solve it".

Reply:
@Lojjs Sure, I think that would be a good idea. I've added "No better allocation is currently possible." Hopefully this indicates that nothing went wrong and if they tried again then they'd get the same outcome. I'm inclined to stay clear of saying things are "optimal" as it probably a bit of a loaded term/require too much explaining 😁 

@LinneaAndersson definitely I agree, I've added 2 examples. The first is where databases are allocated to all servers so there is not much a reallocate could do here. And the second when things look like they could be "improved" but there are constraints on the servers that prevent databases from moving so you're already in the "best" allocation.

Reply:
Also, I put the 5.13 label on this but I'm aware the cut is tomorrow. There is no pressure to get this in for then, so happy to bump it to 5.14 :)